akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -268,7 +268,7 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
       entityDataIn.ensureSubscription()
     }
 
-    abstract class WaitingForRequest(request: HttpRequest) extends RequestEntityTestSetup {
+    abstract class WaitingForRequest(request: HttpRequest) extends RequestEntityTestSetup with Serializable {
```

Comment:
What's up with this?

Reply:
Oops, will remove. Was a test whether I can get rid of scalatest spamming the console with stack traces.